apps/institution-service/src/aggregatorInstitutions/aggregatorInstitutionEndpoints.ts

```diff
@@ -98,4 +99,10 @@ router.get(
   getAggregatorInstitution as RequestHandler,
 );
 
+router.patch(
+  "/:aggregatorId/:aggregatorInstitutionId",
```

Comment:
Rest API conventions don't usually have 2 path params in a row. What if you get rid of `aggregatorId` then for your aggregator validation you could pull the aggregatorId off the aggregatorInstitutionId. 

If you want to keep the aggregatorId I think Rest convention would be more like `/aggregator/:aggregatorId/aggregatorInstiution/:aggregatorInstitutionId` but that would make the route even more ugly because the full route would have "aggregatorInstitution" twice. To me it seems like removing `:aggregatorId` from the path is the way to go.